pkgs/applications/networking/maestral/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, python3Packages, fetchFromGitHub
+, withGui ? false }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "maestral${lib.optionalString withGui "-gui"}";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "SamSchott";
+    repo = "maestral-dropbox";
+    rev = "v${version}";
+    sha256 = "1nfjm58f6hnqbx9xnz2h929s2175ka1yf5jjlk4i60v0wppnrrdf";
+  };
+
+  disabled = python3Packages.pythonOlder "3.6";
+
+  propagatedBuildInputs = (with python3Packages; [
+    blinker click dropbox keyring keyrings-alt requests u-msgpack-python watchdog
+  ] ++ lib.optional withGui pyqt5);
+
+  postInstall = ''
+    makeWrapper $out/bin/maestral $out/bin/maestral-gui \
+      --add-flags gui
```

Comment:
Should this only be added when `withGui` is true

Reply:
Fixed

Reply:
I can reproduce @alexarice's failure with `nix-shell --pure`. The failure is due to using `makeWrapper` here; it should be `makeQtWrapper` or `makeWrapper ... ${qtWrapperArgs[@]}`.

Reply:
Damn those impurities.... Thanks @ttuegel. Fixed.